src/controllers/reviewController.js

```diff
@@ -125,14 +125,14 @@ const getSingleReview = async (req, res) => {
 
 // delete review
 const deleteReview = async (req, res) => {
-  const { reviewId } = req.params; // take the id of the review from req.params
+  const { id } = req.params; // take the id of the review from req.params
```

Comment:
Ah I see now after looking at your frontend ticket; this endpoint wasn't working because it wasn't correctly destructuring the path parameter, good catch!

We could reduce this to just a single-line change by making this: `const { id: reviewId } = req.params;` that way the rest of the function could be left as it originally was, referring to the _variable_ `reviewId` that we've destructured here. Only a suggestion, either way is fine.